mautic/mautic
# Add reply to to custom email

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

This PR add reply to for custom emails.

![image](https://user-images.githubusercontent.com/462477/199737314-0a80d376-0072-4721-9ffc-6fcc59fea016.png)




#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2.  Open contact and send email with reply to
3.  Check If works properly




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11662"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
